### ISSUE TITLE
Start VS code with Julia repl running, and the environment initialized

### ISSUE THREAD

**Issue by PetrKryslUCSD** [julia-vscode/julia-vscode#1622]
When there is a Manifest.toml (and a Project.toml) in the root of the folder that you open with VS Code, it should automatically fire up the REPL, activate and instantiate the project. At least it should be an option.

**Comment by PedroPizarro** [comment]
If you put this:

`using Pkg
if isfile("Project.toml") && isfile("Manifest.toml")
    Pkg.activate(".")
end `

In your satartup.jl, if there is a Project.toml and a Manifest.toml in the current folder, it's going to be activated.
So if you open up a folder in your Code and run julia in the terminal, the project will be activated.

I think there is a way of doing a statement to instantiate the project if necessary,  but I've never though about it and searched for.

I don't know if this will help, because you still need to do a lot of stuff (open code, open the terminal and run julia) to do what you are asking for (if a understood correctly), but I use this config. and I like it.

**Comment by PetrKryslUCSD** [comment]
Good thinking. It precisely fits my current need! (I want students to work with the IDE with the least amount of obstacles and things to do.)

**Comment by davidanthoff** [comment]
I don't think that code should be needed: the extension already checks whether there is a `Project.toml` and `Manifest.toml` in the root folder, and if there is it will activate it.

So we'll just track adding some option to automatically start the REPL at startup in this issue here.

**Comment by logankilpatrick** [comment]
> I don't think that code should be needed: the extension already checks whether there is a Project.toml and Manifest.toml in the root folder, and if there is it will activate it.

This does not seem to be the case, I have a project and manifest file, when I open Julia, it is not activating the environment automatically.
